corehq/apps/users/models.py

```diff
@@ -2473,10 +2476,9 @@ def to_ota_restore_user(self, domain, request_user=None):
             request_user=request_user
         )
 
-    def get_owner_ids(self, domain):
-        owner_ids = [self.user_id]
-        owner_ids.extend(loc.location_id for loc in self._get_case_owning_locations(domain))
-        return owner_ids
+    def get_case_sharing_groups(self, domain=None):
+        assert domain  # fail loudly
```

Comment:
I only see a handful of places that call `get_case_sharing_groups`, would it be possible to make this a required argument?